src/main/java/com/haxwell/apps/quizki/controllers/VerifyCredentialsController.java

```diff
@@ -66,6 +72,19 @@ public User method1(HttpServletRequest request, Model model) throws Exception {
 		return user != null ? user : null;
 	}
 	
+	/*
```

Comment:
This should be handled in corsFilter.java... In fact, it should never get here, because corsFilter returns when the HTTP method is OPTIONS, and corsFilter is run before this code..

Reply:
Working on moving this to the filter. Initially (before the access changes) the request was not making it to the `corsFilter` so I put this code here to handle the `OPTIONS` method.